profile/profile.go

```diff
@@ -1,13 +1,17 @@
 package profile
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"path/filepath"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
+	"github.com/safing/portbase/utils/osdetail"
+
 	"github.com/tevino/abool"
 
 	"github.com/safing/portbase/config"
```

Comment:
Looks like `gofmt` did not reorder the imports. the `safing/portbase` imports belong together